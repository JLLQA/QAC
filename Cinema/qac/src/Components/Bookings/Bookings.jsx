import BookingDetails from './BookingDetails/BookingDetails';

const Bookings = () => {

    return (
        <div>
            <BookingDetails />
            <br />
            <br />
            <br />
            <br />
        </div>
    );
}

export default Bookings;