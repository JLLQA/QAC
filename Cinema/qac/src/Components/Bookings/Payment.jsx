import Navbar from '../Multipage/Navbar/Navbar';

const Payment = () => {


    return (
        <>
            <Navbar />
            <div id="dropped-box" className="container-fluid">
                <h1>Payment</h1>
            </div>
        </>
    );
}

export default Payment;