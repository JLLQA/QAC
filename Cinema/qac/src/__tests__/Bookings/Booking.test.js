import '@testing-library/jest-dom';
import '@testing-library/user-event';
import { render, screen } from '@testing-library/react';
import React from 'react';
import renderer from 'react-test-renderer';
import Bookings from '../../Components/Bookings/Bookings';
import axios from 'axios';

test('bookings text is present', async () => {
    render(<Bookings />);
    await axios({
        method: "Get",
        url: "http://localhost:5000/movies",
        headers: { "Access-Control-Allow-Origin": "*" }
    });
    const bookingText = screen.getByText("BOOKINGS");
    expect(bookingText).toBeInTheDocument();
});

test('bookings loading text is present', () => {
    render(<Bookings />);
    const linkElement = screen.getByText("Loading...");
    expect(linkElement).toBeInTheDocument();
});

test("It matches the snapshot", () => {
    const tree = renderer.create(<Bookings />).toJSON();
    expect(tree).toMatchSnapshot();
})