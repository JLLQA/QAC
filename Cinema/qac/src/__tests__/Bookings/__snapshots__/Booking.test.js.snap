// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It matches the snapshot 1`] = `
<div>
  <div
    className="container-fluid"
    id="dropped-box"
  >
    <h1>
      Loading...
    </h1>
  </div>
</div>
`;
